rosettanet/2.0/modules/ROOT/pages/rosettanet-connector-examples.adoc

```diff
@@ -0,0 +1,486 @@
+
+=== Create the First Flow of the Buyer App
+
+The first flow of the Buyer app receives a purchase order request document via an `HTTP POST` and sends this as a RosettaNet action message to the Seller app. Configuring this flow involves creating a new Mule project, configuring the *HTTP Listener* source, adding a *Send action* operation, and adding a *Transform* component.
+
+==== Configure the HTTP Listener Source
+
+Configure the *HTTP Listener* source to initiate a Mule flow when a call is made to the default path (`/`):
+
+. Create a new Mule project in Studio.
+. F1rom the *Mule Palette* view, select *HTTP* and drag the *Listener* component to the canvas.
+. In the properties window, click *+* next to the *Connector configuration* field to add a global element.
+. In the *Name* field, enter `HTTP_InputListener`.
+. Accept the defaults.
+
+==== Add the PO_InitiatorConfig_Buyer Global Element 
+
+Add the `PO_InitiatorConfig_Buyer` global element for the *Send Action* operation:
+
+. In the properties window for the *Send action* operation, click *+* next to the *Module configuration* field to add a global element.
+. In the *RosettaNet Config* window, complete the following fields:
+
+[%header%autowidth.spread]
+|===
+|Global HTTP Listener|HTTP_Listener-config
+|Service Endpoint Path|/partner1
+|Global HTTP Request|HTTP_Request_configuration
+|Request Endpoint Path|/partner2
+|Keystore Path |/partner1.jks
+|Keystore Access password |nosecret
+|Signing key password |partner1
+|Global usage code |Test
+|Mule DUNS identifier|123456789
+|Mule location identifier | partner1
+|Partner DUNS identifier | 123456788
+|Partner location identifier | partner2
+|role in PIP | INITIATOR
+|PIP definition path | /PIP3A4/V02_02.xml
+|===
++
+. Click *OK*.
+
+==== Add the Send Action Operation
+
+Add the *Send Action* operation to send the purchase order request message to the Seller app by using the configuration defined for the `PO_InitiatorConfig_Buyer` global element:
+
+. From the *Mule Palette* view, select *RosettaNet* and drag the  *Send action* operation next to *Listener*.
+. In the properties window, complete the following fields:
++
+[%header%autowidth.spread]
+|===
+|Field|Value
+|Display Name|Send action
+|Module configuration|PO_InitiatorConfig_Buyer
+|===
+
+==== Add a Transform Message Component
+
+Add a *Transform Message* component to transform the message ID in the purchase order request message to plain text:
+
+. From the *Mule Palette* view, select *Core* and drag a *Transform Message* component next to *Send action*.
+. In the properties window, overlay the brackets in the *Output* section with this DataWeave code:
++
+[source,dataweave,linenums]
+----
+%dw 2.0
+output text/plain
+---
+"Buyer sent action message " ++ attributes.messageId ++ "\n"+
+----
+
+=== Create the Second Flow of the Buyer App
+
+*Juan, please check if this description is correct.*
```

Comment:
Remember to delete this, or put it in comment format so it doesn't show up in the output.